Auth.php

```diff
@@ -32,6 +37,7 @@ public function __construct(Model $userModel = null)
         }
 
         $this->userModel = $userModel;
+        $this->fallbackAuth = new \Piwik\Plugins\Login\Auth();
```

Comment:
Was only curious :) Didn't see the advantage immediately and had to think about it for a while :) I'd have used the `parent::__construct` just to not have to adjust anything in case `Login\Auth` ever gets changed but probably not gonna by changed any time soon anyway. Good to merge :)